source/gui/addonGui.py

```diff
@@ -724,8 +736,13 @@ def __init__(
 		button.Bind(wx.EVT_BUTTON, self.onClose)
 
 		self.refreshAddonsList()
-		self.addonsList.SetFocus()
+		self.SetMinSize(mainSizer.GetMinSize())
+		# Historical initial size, result of L{self.addonsList} being (550, 350) as of 1364839447.

```

Comment:
Also here

Reply:
> Also here

This one is a copy/paste mistake, I'll fix it.